components/automate-chef-io/content/docs/ldap.md

```diff
@@ -30,10 +30,10 @@ Chef Automate does not support using _two_ SAML IdPs or _two_ LDAP services simu
 {{% /warning %}}
 
 Switching between a Microsoft AD configuration and generic LDAP configuration will
-not affect your [policies]({{< relref "authorization-overview.md" >}}), as they are both LDAP configurations.
+not affect your policies, as they are both LDAP configurations.
 However, switching between either of those configurations and a SAML configuration will
-require you to remake your policies, as the [subjects]({{< relref "authorization-overview.md#structure" >}}) of those
-policies will be incorrectly labelled as `ldap`.
+require you to remake your policies if using IAM v1, as the [subjects]({{< relref "iam-v1-overview.md#structure" >}}) of those
+policies will be incorrectly labeled as `ldap`.
```

Comment:
again, old stuff, but it seems like the "label" isn't really the problem, it's the categorization or something. Oh, well.

Reply:
i dont understand, i dont really know anything about iam v1, can you give me a bit more details? or a suggestion of what to say instead?

Reply:
I was trying to say that the phrase "will be labeled incorrectly" suggests that things will _look_ wrong, when really they _won't work_. 

I'm still stuck trying to think of a better way to phrase it.